components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2122,13 +2126,26 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
             },
         );
         if (isCreateStripeCustomerOnUsageEnabled) {
+            let customer: StripeCustomer | undefined;
+            try {
+                customer = (await this.billingService.getStripeCustomer({ attributionId })).customer;
+            } catch (e) {
+                console.error(e);
+            }
+            if (customer) {
```

Comment:
If we shift this into the `try`, we can avoid the if check

Reply:
nope. 
the catch is, that throwing `ResponseError` is a kind of response, which would otherwise be caught.